gpflow/experimental/check_shapes/config.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2022 The GPflow Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Code for configuring check_shapes."
+"""
+from contextlib import contextmanager
+from enum import Enum
+from typing import Iterator, Union
+
+_enabled = True
+
+
+class DocstringFormat(Enum):
+    SPHINX = "sphinx"
+    """
+    Rewrite docstrings in the `Sphinx <https://www.sphinx-doc.org/en/master/>`_ format.
+    """
+
+    NONE = "none"
+    """
+    Do not rewrite docstrings.
+    """
+
+
+_docstring_format = DocstringFormat.SPHINX
```

Comment:
Similarly as above, would it make sense to also specify this as a [gpflow config](https://github.com/GPflow/GPflow/blob/develop/gpflow/config/__config__.py)?